app/src/main/java/org/mozilla/tv/firefox/ScreenController.kt

```diff
@@ -150,6 +151,17 @@ class ScreenController {
         transaction.commit()
     }
 
+    fun dispatchKeyEvent(keyEvent: KeyEvent, fragmentManager: FragmentManager): Boolean {
+        if (keyEvent.keyCode == KeyEvent.KEYCODE_MENU && keyEvent.action == KeyEvent.ACTION_DOWN) {
+            return handleMenu(fragmentManager)
+        }
+
+        if (currentActiveScreen.value == ScreenControllerStateMachine.ActiveScreen.WEB_RENDER) {
+            if (fragmentManager.webRenderFragment().dispatchKeyEvent(keyEvent)) return true
+        }
```

Comment:
Optional nit: these conditional statements are very simple, but also pretty long, which I find makes them a little hard to read.  I find code like the below to be helpful in this kind of situation.

```kotlin
    fun dispatchKeyEvent(keyEvent: KeyEvent, fragmentManager: FragmentManager): Boolean {
        val menuDown = keyEvent.keyCode == KeyEvent.KEYCODE_MENU && keyEvent.action == KeyEvent.ACTION_DOWN
        if (menuDown) {
            return handleMenu(fragmentManager)
        }

        val webRenderIsActive = currentActiveScreen.value == ScreenControllerStateMachine.ActiveScreen.WEB_RENDER
        if (webRenderIsActive) {
            if (fragmentManager.webRenderFragment().dispatchKeyEvent(keyEvent)) return true
        }
        return false
    }
```